workspaces/global-header/app-config.yaml

```diff
@@ -63,14 +72,36 @@ techdocs:
     type: 'local' # Alternatives - 'googleGcs' or 'awsS3'. Read documentation for using alternatives.
 
 auth:
+  environment: development
   # see https://backstage.io/docs/auth/ to learn about auth providers
   providers:
     # See https://backstage.io/docs/auth/guest/provider
     guest: {}
+    github:
+      development:
+        clientId: ${GITHUB_CLIENT_ID}
+        clientSecret: ${GITHUB_CLIENT_SECRET}
+        signIn:
+          resolvers:
+            - resolver: usernameMatchingUserEntityName
 
 scaffolder:
   # see https://backstage.io/docs/features/software-templates/configuration for software template options
 
+# notifications plugin config
+notifications:
+  processors:
+    email:
```

Comment:
Couldn't we live without email processor? But it's fine. I hope this doesnt' connect always to gmail and tries to send emails???

Reply:
It will cause this error when backend starts

```
backend:start: ForwardedError: Module 'email' for plugin 'notifications' startup failed; caused by Error: Missing required config value at 'notifications.processors.email' in 'app-config.local.yaml'
```

Reply:
Of couse, but you can simple drop `@backstage/plugin-notifications-backend-module-email` from `packages/backend/package.json` and `packages/backend/src/index.ts`